src/NewCommand.php

```diff
@@ -465,7 +465,7 @@ protected function promptForDatabaseOptions(string $directory, InputInterface $i
                 default: $defaultDatabase
             );
 
-            if ($database !== $defaultDatabase) {
+            if ($database !== $defaultDatabase && $this->hasMariaDBConfig($directory)) {
```

Comment:
@nunomaduro I wonder if we should add a new `isLaravel11OrHigher` method or something similar to better indicate the reason for this check?